packages/account/src/Components/poi/poi-form-on-signup/idv-doc-submit-on-signup/idv-doc-submit-on-signup.jsx

```diff
@@ -295,7 +304,7 @@ export const IdvDocSubmitOnSignup = ({ citizen_data, has_previous, onPrevious, o
                                                                     autoComplete='off'
                                                                     placeholder='Enter your document number'
                                                                     value={values.document_number}
-                                                                    onPaste={e => e.preventDefault()}
+                                                                    onPaste={e => preventEmptyClipboardPast(e)}
```

Comment:
```suggestion
                                                                    onPaste={preventEmptyClipboardPast}
```
@yauheni-deriv nit-pick🙂

Reply:
cool, thanks a lot